brainkeeper/src/app/components/session_screen/big-person-card/big-person-card.component.html

```diff
@@ -0,0 +1,31 @@
+<div *ngIf="isRight; else isWrong">
+  <div *ngIf = "show; else open()">
+    <div *ngIf = "chosenPerson" class = "big-card">
+      <mat-card class="big-card-right" id = "bcardr" >
+          <mat-card-header>
+            <mat-card-title>{{chosenPerson.name}}</mat-card-title>
+          </mat-card-header>
+          <img mat-card-image src="{{'data:image/jpg;base64,' + chosenPerson.picture}}" alt="Photo of the selected Person" class = "mat-card-image">
+          <mat-card-actions>
+            <button mat-button (click)= "close()">WEITER</button>
+          </mat-card-actions>
```

Comment:
Please use english language for button texts

Reply:
changed with commit: 3c092315bd02c68198ed286b95baee75aafa7339